### ISSUE TITLE
Cannot read property 'style' of undefined

### ISSUE THREAD

**Issue by garyrutland** [sbugert/react-native-admob#348]
Hi,

Trying to get AdMob working in a project and followed the steps to install but when ever I import this library using `import { AdMobBanner } from 'react-native-admob'` I get the error `Cannot read property 'style' of undefined`.

If I try and use the beta version I get a different error `Invariant Violation: requireNativeComponent: "RNGADBannerView" was not found in the UIManager`.

Any ideas what the problem is?

Thanks,
Gary

**Comment by Verma-Pooja** [comment]
Cannot read property 'style' of undefined
May In Code, you misplace or incorrectly spelling use of style.
Invariant Violation: requireNativeComponent: "RNGADBannerView" was not found in the UIManager.
Solution is: link your admob. i.e react-native link react-native admob.
you can refer to this link:
https://react-native-solutions.blogspot.com/2018/08/problems-related-to-admob-updated.html

**Comment by garyrutland** [comment]
I did try numerous times to check the spelling of `style` but I wasn't actually referencing it anywhere.
That issue only appeared when I merely imported the library, not even using it.

Regarding the second error, I ran `react-native link` numerous times and would see the following but still got the error.
```
rnpm-install info Platform 'ios' module react-native-admob is already linked
rnpm-install info Platform 'android' module react-native-admob is already linked
```

**Comment by Verma-Pooja** [comment]
@garyrutland, you did everything right.
can you send the piece of code and which RN version you using?

**Comment by garyrutland** [comment]
Any update on this at all?

**Comment by snapxtp** [comment]
@garyrutland  I do not think so. Because I tried everything but the same error continues and this plugin never worked :(

**Comment by garyrutland** [comment]
@sbugert Is anyone actively working on this project anymore?
It used to work but doesn't seem to anymore.

**Comment by berkantunal** [comment]
Problem in React.propTypes you can import prop-types and change AdMobBanner.propTypes and RNPublisherBanner.propTypes 

```
import PropTypes from 'prop-types';

React.PropTypes.string -> PropTypes.string,
```

**Comment by AMINDRH91** [comment]
Ans ===> https://github.com/sbugert/react-native-admob/issues/397#issuecomment-454407733
